common/cards/card-plugins/single-use/potion-of-slowness.js

```diff
@@ -9,22 +13,31 @@ class PotionOfSlownessSingleUseCard extends SingleUseCard {
 			name: 'Potion of Slowness',
 			rarity: 'common',
 			description:
-				"Opponent's active hermit can only use their primary attack on their next turn.\n\nDiscard after use.",
+				"Opponent's active hermit can only use their primary attack on their next turn.",
 		})
 	}
 
+	canApply() {
+		return true
+	}
+
 	/**
 	 * @param {GameModel} game
+	 * @param {string} instance
+	 * @param {CardPos} pos
 	 */
-	register(game) {
-		game.hooks.applyEffect.tap(this.id, () => {
-			const {singleUseInfo, opponentActiveRow} = game.ds
-			if (singleUseInfo?.id === this.id) {
-				if (opponentActiveRow === null) return 'INVALID'
-				opponentActiveRow.ailments.push({id: 'slowness', duration: 1})
-				return 'DONE'
-			}
-		})
+	onApply(game, instance, pos) {
+		const {otherPlayer} = pos
+		const opponentActiveRow = getActiveRow(otherPlayer)
+		if (!opponentActiveRow) return
+		opponentActiveRow.ailments.push(
+			{id: 'slowness', duration: 1},
+			{id: 'weakness', duration: 1},
+			{id: 'fire', duration: 1},
+			{id: 'poison', duration: 1},
+			{id: 'badomen', duration: 1},
+			{id: 'sleeping', duration: 1}
```

Comment:
Looks like you were testing some stuff here.. but we don't want every ailment to come from the weakness potion!

Reply:
oops, yes, I was testing the status icons and I forgot to undo it, may bad.